TweetCell.h

```diff
@@ -17,16 +9,13 @@ NS_ASSUME_NONNULL_BEGIN
 @property (weak, nonatomic) IBOutlet UILabel *screenNameLabel;
 @property (weak, nonatomic) IBOutlet UILabel *dateLabel;
 @property (weak, nonatomic) IBOutlet UILabel *tweetTextLabel;
-@property (weak, nonatomic) IBOutlet UILabel *replyCountLabel;
 @property (weak, nonatomic) IBOutlet UILabel *retweetCountLabel;
 @property (weak, nonatomic) IBOutlet UILabel *favoriteCountLabel;
 @property (weak, nonatomic) IBOutlet UIImageView *profilePictureView;
-@property (weak, nonatomic) IBOutlet UIImageView *shareView;
-@property (weak, nonatomic) IBOutlet UIImageView *retweetView;
-@property (weak, nonatomic) IBOutlet UIImageView *likeView;
+@property (weak, nonatomic) IBOutlet UIButton *likeView;
```

Comment:
This is why I hate the naming convention of saying what something is in the name itself. This is a button now but the compiler won't tell you that there's a mismatch between var name and type. A reader will be confused that this says `View` but is `Button`.